src/main/resources/idea/pmd-settings.xml

```diff
@@ -0,0 +1,9 @@
+<component name="PDMPlugin">
+    <option name="options">
+        <map>
+            <entry key="Encoding" value="" />
```

Comment:
Used on rev projects, but i didn't found this entry as required, by default pmd plugin for idea used utf-8, and I think it is enough